governance/xc_admin/packages/xc_admin_frontend/components/tabs/UpdateProductMetadata.tsx

```diff
@@ -0,0 +1,346 @@
+          setIsSendProposalButtonLoading(false)
+        } catch (e: any) {
+          toast.error(capitalizeFirstLetter(e.message))
+          setIsSendProposalButtonLoading(false)
+        }
+      }
+    }
+  }
+
+  const ModalContent = ({ changes }: { changes: any }) => {
+    return (
+      <>
+        {Object.keys(changes).length > 0 ? (
+          <table className="mb-10 w-full table-auto bg-darkGray text-left">
+            <thead>
+              <tr>
+                <th className="base16 py-8 pl-6 pr-2 font-semibold lg:pl-6">
+                  Description
+                </th>
+                <th className="base16 py-8 pl-1 pr-2 font-semibold lg:pl-6">
+                  Value
+                </th>
+              </tr>
+            </thead>
+            {Object.keys(changes).map((key) => {
+              const { prev, new: newProductMetadata } = changes[key]
+              const diff = Object.keys(prev).filter(
+                (k) => prev[k] !== newProductMetadata[k]
+              )
+              return (
+                <tbody key={key}>
+                  {diff.map((k) => (
+                    <tr key={k}>
+                      <td className="base16 py-4 pl-6 pr-2 lg:pl-6">
+                        {k
+                          .split('_')
+                          .map((word) => capitalizeFirstLetter(word))
+                          .join(' ')}
+                      </td>
+                      <td className="base16 py-4 pl-1 pr-2 lg:pl-6">
+                        {newProductMetadata[k]}
+                      </td>
+                    </tr>
+                  ))}
+                </tbody>
+              )
+            })}
+          </table>
+        ) : (
+          <p className="mb-8 leading-6">No proposed changes.</p>
+        )}
+        {Object.keys(changes).length > 0 ? (
+          !connected ? (
+            <div className="flex justify-center">
+              <WalletModalButton className="action-btn text-base" />
+            </div>
+          ) : (
+            <button
+              className="action-btn text-base"
+              onClick={handleSendProposalButtonClick}
+            >
+              {isSendProposalButtonLoading ? <Spinner /> : 'Send Proposal'}
+            </button>
+          )
+        ) : null}
+      </>
+    )
+  }
+
+  // create anchor wallet when connected
+  useEffect(() => {
+    if (connected) {
+      const provider = new AnchorProvider(
+        connection,
+        anchorWallet as Wallet,
+        AnchorProvider.defaultOptions()
+      )
+      setPythProgramClient(
+        pythOracleProgram(getPythProgramKeyForCluster(cluster), provider)
+      )
+    }
+  }, [anchorWallet, connection, connected, cluster])
+
+  return (
+    <div className="relative">
+      <Modal
+        isModalOpen={isModalOpen}
+        setIsModalOpen={setIsModalOpen}
+        closeModal={closeModal}
+        content={<ModalContent changes={productMetadataChanges} />}
```

Comment:
Modal doesn't show which Symbol you are modifying

Reply:
good catch